applications/nrf_desktop/description.rst

```diff
@@ -1048,6 +1048,24 @@ See the following list of possible scenarios and best practices:
   By default, the :kconfig:option:`CONFIG_USB_HID_POLL_INTERVAL_MS` Kconfig option is set to ``1`` to request the lowest possible poll interval.
```

Comment:
It is only for legacy stack IIRC, we should also mention next stack here I think and the method of configuring it through DTS.

Maybe we should redirect to the `Low latency device configuration` of the usb_state instead here so the method of configuration is only in one place?

Reply:
Please keep in mind that `Low latency device configuration` is currently a subsection of `USB HID configuration in USB legacy stack`

Reply:
Oh okay, then maybe such section should also appear in the USB next stack?